### DISCUSSION TITLE
Calling membership

### DISCUSSION THREAD

**Discussion by carlbannister** [liedekef/events-made-easy#311] [Q&A] [answered]
How can i use a custom field within a membership application form to call a certain membership.

I have created 3 different memberships senior, junior, committee. I have created a membership form and put a custom dropdown field Senior||Junior||Committee and would like to use this to select the relevant membership to be applied for.

Currently i call a separate form for each membership but effectively they are all the same 

Thanks

**Comment by liedekef** [comment]
The decision on which membership is wanted should indeed be done on a separate page.
If they are the same, why create different memberships? A custom field can be created for members (but it will never allow to select a different membership), and one could link a discount to it, but I'd keep the different memberships if that's too complicated.

**Reply by carlbannister** [reply]
Ok thats clear, reason i have done it like this is that each membership is at a different price. At the time i originally set it up it was the easiest way with my limited knowledge, maybe there is a better way of setting up memberships?

**Reply by liedekef** [reply, marked as answer]
You could use a discount by creating a custom field with purpose "members"n type "dropdown" and setting "Extra charge" to "yes". Then the dropdown values ("Field values") should reflect the price difference (can be negative) and the "Field tags" can reflect your membership subtype names.
That field should then be used in your membership signup form of course.
